zookeeper-api/src/main/java/org/apache/helix/zookeeper/zkclient/ZkClient.java

```diff
@@ -1074,6 +1101,34 @@ private Stat getStat(final String path, final boolean watch) {
     }
   }
 
+  /*
+   * Install watch if there is such node and return the stat
+   * Don't install watch if there is no such node and return null
+   *
+   * Use ZooKeeper native api getData() as underlying method
+   */
+  private Stat getStat2(final String path) {
```

Comment:
Can we use a better name than getStat2?

Reply:
getStatNoInstallingWatchWhenNodeNotExists()? this seems to be too long. 
Or maybe getStatNoWatchNoNode()? 
Any better name?

Reply:
changed.